plugin/targets.vim

```diff
@@ -16,9 +16,16 @@ function! s:addAllMappings()
     let g:targets_nl   = get(g:, 'targets_nl', get(g:, 'targets_nlNL', 'nl')[0:1]) " legacy fallback
     let g:targets_aiAI = get(g:, 'targets_aiAI', 'aiAI')
```

Comment:
That's what I initially wanted to do, but then if a user sets `g:targets_nl` only after the plugin was loaded, their new settings won't be parsed correctly.
What do you think?

Reply:
Yes that's true. But I think it's fine. And is already the case for existing settings. Take `g:targets_aiAI` for example. In order to make that dynamic we'd need to check it for having changed and then update our mappings accordingly. I think (at least for now) it's fine to ask users to restart their Vim so their new settings take effect.

We might reconsider this and see if we can make this more dynamic, but I don't think we should do that in this PR :v:

So I'd still like to suggest to avoid calling autoload functions in `plugin/` code.

Reply:
@infokiller: Thanks! There's still this comment which hasn't been addressed yet.

Reply:
Sorry, missed that!
Should be addressed now.